package.json

```diff
@@ -66,6 +66,7 @@
     "*.{js,json,ts,tsx,css,md,mdx}": "prettier --write"
   },
   "resolutions": {
-    "rollup": "^2.33.3"
+    "rollup": "^2.33.3",
+    "**/update-notifier/boxen": "^5.0.0"
```

Comment:
why not wait for your update-notifier PR to land?

Reply:
I think the real fix we need is in the DefinitelyTyped PR which currently has CI failing - so this is a workaround for now to get them all in line.